libs/expo/betterangels/src/lib/helpers/generatePublicNote.ts

```diff
@@ -9,7 +9,7 @@ interface IWatchedValue {
 }
 
 export default function generatePublicNote(watchedValues: IWatchedValue) {
-  const { purposes, moods, providedServices, nextSteps, requestedServices } =
```

Comment:
Hey @Davit-BetterAngels 
I noticed there is underscore between the service name if they are two words or more.

<img src="https://github.com/user-attachments/assets/8a8ef05f-91c1-4854-aa3e-8c2e537025db" width="360" />

Reply:
fixed

Reply:
Cool, thanks! I confirmed it was fixed! Thanks so much!